Add example for multiple variable usage (#21589)
DataDog/documentation: content/en/dashboards/guide/context-links.md

Remapping the IP address creates a context link that connects your RUM events with corresponding logs.

### Configure multiple variables

The following example explains how to configure multiple variables and conditions in your context link query.

#### Context

Add context links to investigate specific logs or conditions. 
- You have multiple tag values with the same context (for example, `env:production OR env:prod`). 
- You want to filter down logs to multiple conditions (for example, `env:prod AND service:backend`)

#### Approach

After you select the template variables you want to troubleshoot, the context link configuration takes those template variables and inserts them into the query. **Note**: The syntax and the parenthesis enclosure impacts the query. 

> As an example, you want to configure a context link with the `service:backend` AND (`env:production` OR `env:prod`). Use the following configuration:

Reviewer: ```suggestion
For example, if you want to configure a context link with `service:backend` AND (`env:production` OR `env:prod`), use the following configuration:
```

```
service:backend (env:{{$env.value}})
```

#### Result

> The parenthesis translates the `(env:{{$env.value}})` to (env:*) which allows you to enter multiple variables into your context links query.

Reviewer: ```suggestion
The parenthesis translates the `(env:{{$env.value}})` to `(env:*)` which allows you to enter multiple variables into your context links query.
```
